docs/sdk-guide.md

```diff
@@ -766,6 +771,39 @@ public VectorSchemaRoot getArrowVectors() throws IOException;
 public static ArrowRecordBatch byteArrayToArrowBatch(byte[] batchBytes, BufferAllocator bufferAllocator) throws IOException;
 ```
 
+### Class org.apache.carbondata.sdk.file.PaginationCarbonReader
+```
+/**
+* Pagination query with from and to range.
+*
+* @param from must be greater than 0 and <= to
+* @param to must be >= from and not outside the total rows
+* @return array of rows between from and to (inclusive)
+* @throws Exception
+*/
+public Object[] read(long from, long to) throws IOException, InterruptedException;
+```
+
+```
+/**
+* Get total rows in the folder.
```

Comment:
Suggestion: Get total rows in the folder or a list of CarbonData files

Reply:
done